src/v/coproc/event_handler.cc

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
```
Add event_handler for process events with coproc_type nodejs or data-…
…policy
```

Avoid long lines in commit messages. The standard lengths are specified in CONTRIBUTING.md. I think they are 50/72 for subject and body.

Reply:
done